### ISSUE TITLE
Add staging in volcano.sh/volcano for api subproject

### ISSUE THREAD

**Issue by william-wang** [volcano-sh/volcano#2356]
#### What would you like to be added:
Add volcano/staging directory for api subproject and make code automatically synchronazation.

#### Why is this needed:
It's inconvinient for dev to submit pr to api repo and volcano repo seperately.

**Comment by hwdef** [comment]
I think this still needs
/reopen

**Comment by srikanth-iyengar** [comment]
Hello @hwdef, I'd like to inquire if I could select this particular issue as my initial task for contributing to Volcano?

**Comment by hwdef** [comment]
> Hello @hwdef, I'd like to inquire if I could select this particular issue as my initial task for contributing to Volcano?

I don't think this is appropriate because the most important thing about this problem is that we need a robot to synchronize the two repo. This job should be handled by Huawei

**Comment by srikanth-iyengar** [comment]
Ohk Thanks for the info

**Comment by Monokaix** [comment]
> Ohk Thanks for the info

I think there is still something we can do here. We can refer to the repos synchronization method of k8s, we should figure out the specific implementation strategy and move the api code here.
